src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1092,9 +1092,6 @@ InferenceProfiler::Measure(
     if (include_server_stats_) {
       RETURN_IF_ERROR(GetServerSideStatus(&start_status));
     }
-    // Need to zero out start stats when window start is 0
```

Comment:
This undoes the changes in https://github.com/triton-inference-server/client/pull/150

Can someone explain why that was wrong or why this fixes another issue?

Reply:
It was wrong, and was the root cause of this bug. The true fixes to the previous issue were in https://github.com/triton-inference-server/client/pull/195 and https://github.com/triton-inference-server/client/pull/191.

Reply:
At the start of the first window we need to sync the server and client stats, so we have a "starting point". Then at the end of each window we can sync again to see how much it has changed, and thus what happened during that window.  

This code was incorrectly setting the values to 0 at the start of the first window. That meant that if the server had already seen 1000 requests before this run of PA, then at the end of the first window it would appear that 1000 extra requests were sent in that first window.

Reply:
Oh interesting. This seems big.